### DISCUSSION TITLE
Welcome to UTN_BsAs_Tercer_Semestre Discussions!

### DISCUSSION THREAD

**Discussion by vmgith** [CodeSystem2022/UTN_BsAs_Tercer_Semestre#33] [Announcements]
Tenemos que coordinar el dia para la proxima reunion x Discord

**Comment by mchalin** [comment]
Yo puedo hoy por la tarde

**Comment by alealgarin92** [comment]
Yo tambien puedo

**Comment by vmgith** [comment]
REunion realizada con todos los integrantes. Debatimos como subir la semana abierta y revisamos el repositorio.
